crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -513,8 +513,8 @@ mod tests {
                 .await;
                 let cell_network = test_network.cell_network();
                 let (tx_complete, rx_complete) = tokio::sync::oneshot::channel();
-                // We are expecting six ops per agent plus one for self plus 7 for genesis.
-                let total_expected = (num_agents + 1) * (6 + 7);
+                // We are expecting six ops per agent plus one for self.
+                let total_expected = (num_agents + 1) * 6;
```

Comment:
Did this change because we're now filtering out recent publishes? If so then this comment is helpful.

```suggestion
                // We are expecting six ops per agent plus one for self.
                // The 7 genesis ops were already recently published, so
                // won't be published again this time.
                let total_expected = (num_agents + 1) * 6;
```